apps/docs/app/pages/getting-started/individual-packages/page.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import { allPages } from "contentlayer/generated";
+import Mdx from "@/components/mdx/Mdx";
+import { notFound } from "next/navigation";
+import Aside from "@/app/ui/layout/aside/Aside.tsx";
+import getSectionLinks from "@/app/lib/getSectionLinks";
+
+export default function GettingStartedPage() {
+    const page = allPages.find(gettingStartedPage => gettingStartedPage._id === "pages/individual-packages.mdx");
+
+    if (!page) {
+        notFound();
+    }
+
+    const sectionLinks = getSectionLinks(page);
+
+    return (
+        <div className="hd-wrapper hd-flex">
+            <Aside title="On this page" links={sectionLinks} />
+            <main>
+                <article className="hd-content" key={page._id}>
+                    <h1 className="hd-title hd-title--level1">{page.title}</h1>
```

Comment:
The `Title` component exists for this use case. 

`<Title as="h1">{title}</Title>`

ref: https://github.com/gsoft-inc/wl-hopper/blob/6cd15ebe9e25582349fb519cd22f01d6e52fa22a/apps/docs/components/title/Title.tsx